OS-SPEC.md

```diff
@@ -7,12 +7,23 @@ This document attempts to define a minimal set of things that must exist for mos
 
 ACIs that define the label `os=linux` can expect this environment by default.
 
+## Execution Environment
+
+It is RECOMMENDED that the following environment variables be set for each application's main process and any lifecycle processes:
```

Comment:
Is there some real requirement driving the desire for these?  I feel like this is soft enough that it doesn't matter at all and should just be stricken entirely.